utils/js/Decorators.ts

```diff
@@ -7,7 +7,7 @@
 import {XH} from '@xh/hoist/core';
 import {debounce, isFunction} from 'lodash';
 import {getOrCreate, throwIf, warnIf} from './LangUtils';
-import {withDebug, withInfo} from './LogUtils';
+import {withDebug, withInfo} from '../log';
```

Comment:
Should we be using the relative imports here, or avoid those and consistently import from the full path?

Reply:
changed to use full path.  My thinking is: imports outside of package should use full path.